deployments/clowder.yaml

```diff
@@ -102,6 +102,20 @@ objects:
                     name: app-secret
               - name: CLIENTS_RBAC_BASE_URL
                 value: "${CLIENTS_RBAC_BASE_URL}"
+              - name: CLIENTS_PENDO_BASE_URL
+                value: "${CLIENTS_PENDO_BASE_URL}"
+              - name: CLIENTS_PENDO_INTEGRATION_KEY
+                valueFrom:
+                  secretKeyRef:
+                    key: integration-key
+                    name: pendo-creds
+                    optional: true
+              - name: CLIENTS_PENDO_TRACK_EVENT_SECRET
```

Comment:
In PR #231 this env var ends with `KEY` and not `SECRET`.  And the other is already name: `CLIENTS_PENDO_API_KEY` but here it is `CLIENTS_PENDO_INTEGRATION_KEY`.

I.e. we have a mismatch and need to unite.

Reply:
Right! Updated description to reference #311 and label! 

Updated!